src/pages/SettingsPage.js

```diff
@@ -151,31 +170,43 @@ const Settings = ({ toRealmExplorer, currentUserPasswordHash, requestStorageWrit
       const fridge2 = createRecord(UIDatabase, 'Location', { description: 'Fridge 2', code: 'f2' });
       const fridge3 = createRecord(UIDatabase, 'Location', { description: 'Fridge 3', code: 'f3' });
       const fridge4 = createRecord(UIDatabase, 'Location', { description: 'Fridge 4', code: 'f4' });
+      const fridge5 = createRecord(UIDatabase, 'Location', { description: 'Fridge 5', code: 'f5' });
 
       // create sensors
       createRecord(UIDatabase, 'Sensor', {
         name: 'sensor 1',
-        macAddress: 'sen1',
+        macAddress: '00:00:00:00:00:01',
         batteryLevel: 10,
         location: fridge1,
+        isActive: false,
```

Comment:
Oh - why are you setting this as false? I just generated the data on this PR and hadn't noticed - I noticed only when I went back to my branch which I am working on which is interacting with `isActive` heh

Reply:
yeah, sorry. I'll revert that. I was using `isActive` to stop the bluetooth actions for fake sensors.